CONTRIBUTING.md

```diff
@@ -4,7 +4,7 @@ Guidelines for contributing to the Roslyn repo.
 
 ## Submitting Pull Requests
 
-- **DO** ensure submissions pass all Jenkins legs and are merge conflict free.
+- **DO** ensure submissions pass all Azure Devops legs and are merge conflict free.
```

Comment:
Should this make it clear that if you're unsure, just submit a draft PR? This otherwise mandates a standard that our own team doesn't meet.

Reply:
I agree that _someone_ should look at our contributing docs... This is a general pain point I feel on the team today: should I even bother running Test.cmd? Seems like most folks just submit PRs because Test.cmd has drifted so far from reality.